src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java

```diff
@@ -213,6 +227,19 @@ public List<Issue> defineTestItemsIssues(ReportPortalUser.ProjectDetails project
     return updated;
   }
 
+  private void saveMakeDecisionStatistics(Long projectId, int itemsAmount) {
+    Project project = projectRepository.findById(projectId).orElseThrow(
+        () -> new ReportPortalException(PROJECT_NOT_FOUND, projectId));
+    AnalyzerConfig analyzerConfig = getAnalyzerConfig(project);
+
+    var analyticsMetadata = new HashMap<String, Object>();
+    analyticsMetadata.put("autoAnalysisOn", analyzerConfig.getIsAutoAnalyzerEnabled());
+    analyticsMetadata.put("userAnalyzed", itemsAmount);
+
+    analyticsStrategyFactory.findStrategy(AnalyticsObjectType.DEFECT_UPDATE_STATISTICS)
+        .persistAnalyticsData(analyticsMetadata);
```

Comment:
let's provide projectId here as a parameter and move all logic according to analyticsMetadata inside analytics strategy